### ISSUE TITLE
Cannot import ReSwiftThunkTesting via SPM

### ISSUE THREAD

**Issue by mhamann** [ReSwift/ReSwift-Thunk#56]
I'm using ReSwiftThunk via SPM. In my test file, I try to import the testing helper like this:

```swift
import ReSwiftThunk
import ReSwiftThunkTesting
```

but when I do, Xcode shows the error `No such module 'ReSwiftThunkTesting'`.

This is an Xcode Project...not a pure Swift module, so not sure if that has anything to do with it.

**Comment by mhamann** [comment]
Of course, as soon as I opened this, I discovered that the updates to the `Package.swift` file to include the testing library haven't been tagged as a proper version. See https://github.com/ReSwift/ReSwift-Thunk/blob/2.0.1/Package.swift

Any chance a v2.0.2 can be released to fix this?

**Comment by mhamann** [comment]
Also, I think perhaps `ReSwiftThunkTesting` needs to declare a dependency on `ReduxThunk`?

**Comment by DivineDominion** [comment]
@mhamann Thanks for reporting this! Could you test the package dependency locally and open a PR if changes need to be made? I can go ahead and create a tag when everything works 👍

**Comment by rei315** [comment]
<img width="500" src="https://user-images.githubusercontent.com/27776755/213078238-26bb1a7e-d5ae-429b-bf57-6b842cc92f59.png" />

Even with the master branch, There was a problem that ReSwift was not imported in the "ExpectThunk.swift"
Is there any update?
